### DISCUSSION TITLE
Heroku or bust?

### DISCUSSION THREAD

**Discussion by queuebit** [stringer-rss/stringer#668] [General]
Where are people hosting stringer these days?
Or are you using something else instead?

I waited for Heroku to tear down my instance before I started thinking about what to do next.
Now is that time I guess.

**Comment by mockdeep** [comment]
I'm still hosting on Heroku. There's a plan that's only $0.01/month, so I upgraded to that. If they unlinked your database, you can put in a support request and ask them to restore it for you. Not sure how long they'll keep the backups around, though. Personally, I'm pretty comfortable on Heroku. Don't have any major complaints that would push me to jump ship. I occasionally look at other solutions, but they don't appear to be quite as complete in terms of features, yet.

**Reply by queuebit** [reply]
Thanks!

That sounds like a pretty good plan.
